mypy/main.py

```diff
@@ -451,6 +451,7 @@ def get_init_file(dir: str) -> Optional[str]:
     'python_version': lambda s: tuple(map(int, s.split('.'))),
     'strict_optional_whitelist': lambda s: s.split(),
     'custom_typing_module': str,
+    'mypy_path': lambda s: [p.strip() for p in s.split(',')],
```

Comment:
Splitting on commas and colons seems reasonable, but we shouldn't split on spaces: it's not uncommon to have spaces in directory names.  It'd be nice to mention allowable path separators in the documentation for this config option.

@gvanrossum do we currently have tests for the config file somewhere?